source/excerptexport/views.py

```diff
@@ -36,10 +37,13 @@ def new_excerpt_export(request):
 
 @login_required(login_url='/admin/')
 def create_excerpt_export(request):
-    viewContext = {}
     if request.POST['form-mode'] == 'existing_excerpt':
         existingExcerptID = request.POST['existing_excerpt.id']
-        viewContext = { 'use_existing': True, 'excerpt': existingExcerptID }
+        viewContext = { 'excerpt': existingExcerptID }
```

Comment:
@wasabideveloper we should get flake8 up an running, anyway (if possible as a git pre-commit hook). That should force us into proper naming habits and a standardized code style.